packages/fxa-payments-server/src/routes/Subscriptions/Reactivate/ConfirmationDialog.tsx

```diff
@@ -97,6 +115,60 @@ export default ({
           </Localized>
         </button>
       </div>
+    </>
+  );
+};
+
+const ConfirmationDialog = ({
+  onDismiss,
+  onConfirm,
+  plan,
+  customer,
+  customerSubscription,
+  periodEndDate,
+}: {
+  onDismiss: Function;
+  onConfirm: () => void;
+  plan: Plan;
+  customer: Customer;
+  customerSubscription: WebSubscription;
+  periodEndDate: number;
+}) => {
+  const { webIconURL, webIconBackground } = metadataFromPlan(plan);
+  const { last4 } = customer;
```

Comment:
Does this need to be pulled out of the customer object like this since it is only used once? Will `customer.last4` on line 159 suffice?